doc/api/fs.md

```diff
@@ -1549,6 +1549,23 @@ Synchronous version of [`fs.readFile`][]. Returns the contents of the `file`.
 If the `encoding` option is specified then this function returns a
 string. Otherwise it returns a buffer.
 
+_Note: The behavior of `fs.readFileSync()` is platform specific. As such,
```

Comment:
Except those methods both implicitly use `fs.open()`/`fs.read()`. So really the note should be above one of those instead, but many people use `readFile()` as well. That is why I suggested possibly putting the note for `readFile()` and `fs.read()` (and their sync counterparts).